src/repo.rs

```diff
@@ -989,6 +1001,36 @@ impl Repository {
         Ok(())
     }
 
+    /// Merges the given commit(s) into HEAD, writing the results into the working directory.
+    /// Any changes are staged for commit and any conflicts are written to the index. Callers
+    /// should inspect the repository's index after this completes, resolve any conflicts and
+    /// prepare a commit.
```

Comment:
Could this included the rest of the docs as well? (e.g. just copied from libgit2), also, can this be wrapped to 80 chars?